src/featuredetection.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module utils/featuredetection
+ */
+
+/**
+ * Holds feature detection used by the editor.
+ *
+ * @protected
+ * @namespace
+ */
+export default {
+	/**
+	 * Indicates whether the current browser supports ES2018 Unicode properties like `\p{P}` or `\p{L}`.
+	 *
```

Comment:
Since I'm always searching for this. Please add a link to:

https://www.unicode.org/reports/tr44/#GC_Values_Table

so we get a link to those categories :)